### PR TITLE
The menu should be closed if the mouse is out of the Status block (an…

### PR DESCRIPTION
**Pull request by mojgan-a**
…ywhere on thfilteredMenue page) /TASS-985

### CODE REVIEW

**Comment by mojgan-a** [comment]
The menu should be closed if the mouse is out of the Status block (anywhere on thfilteredMenue page)

File: src/Nowruz/modules/layout/components/statusDropDown/index.tsx [hunk 1]
```diff
@@ -29,44 +30,54 @@ export const StatusDropDown: React.FC<StatusDropDownProps> = (props) => {
   const handleOutsideClick = (e) => {
     if (newRef.current && !newRef.current.contains(e.target)) { if (open) { setOpen(!open) } }
   }
+  // const handleOutsideClick = (e) => {
```
Comment (bastantebom) [inline src/Nowruz/modules/layout/components/statusDropDown/index.tsx:33]: If we don't need this maybe we can clear this out?

**Review by bastantebom — CHANGES_REQUESTED** [review]
NIT: Kindly clean the not needed block of code

File: src/Nowruz/modules/layout/components/statusDropDown/index.tsx (hunk 1 above)
Comment (mojgan-a) [reply src/Nowruz/modules/layout/components/statusDropDown/index.tsx]: I defined this function to close the menu when we click on the page and now I deleted it.

**Review by bastantebom — APPROVED** [review]

**Pull request merged**